_posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Receive mentorship from Amazon engineers and accelerate your career in Tech"
+authors:
+  - rednaksi
+date:   2023-09-05 09:00:00 -0700
+categories:
+  - announcements
+meta_keywords: Open Source Contributor Initiative, Google Summer of Code, Major League Hacking, MIT Summer Camp, Coding Bootcamp
+
+meta_description: The OpenSearch Project launches Open Source Contributor Initiative (OSCI). Spanning 10 weeks, this fully remote program allows participants to receive mentorship from seasoned engineers within a nurturing and inclusive environment. Participants in OSCI will work with Amazon engineers and will enhance their portfolios by directly contributing to a prominent open-source software project. When fully leveraged, these benefits can culminate in a recommendation letter signed by an AWS Principal Engineer and an internal endorsement.
+---
+
+
+<img src="/assets/media/blog-images/2023-09-05-college-contributor-initiative/Hot Air Balloon.jpg" alt="Aerial Balloon Express"/>{: .img-fluid}
+# Receive mentorship from Amazon engineers and accelerate your career in Tech
+
+Many today find it challenging to navigate the rugged terrain on their journey to a career in Tech. You can’t know how long your chosen path will be, how difficult it will be, or even if it will lead to the desired destination. You might be questioning whether your coursework is genuinely preparing you for roles in major tech companies. In keeping with the theme of the image above, the OpenSearch Project presents an “Aerial Balloon Express” to bridge the skill and experience gap between your current position and your ultimate career destination. 
+
+This proverbial "Aerial Balloon Express" is the Open Source Contributor Initiative (OSCI). Spanning 10 weeks, this fully remote program allows participants to receive mentorship from seasoned engineers within a nurturing and inclusive environment that fosters learning and collaboration. Participants in OSCI will work with Amazon engineers and will enhance their portfolios by directly contributing to a prominent open-source software project. When fully leveraged, these benefits can culminate in a recommendation letter signed by an AWS Principal Engineer and an internal endorsement. Imagine dedicating just a few hours each week for 10 weeks and reaping such immense value in return. The best part is that the OSCI program can fit any schedule (even if you’re working or studying full time).
+
+The deadline for applications is **September 13th**, and the program runs **September 20th–November 29th**.
+**Apply now: https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY**
```

Comment:
you'll want to update so this is hyperlinked: 
`[https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY](https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY)`